HISTORY.md

```diff
@@ -4,13 +4,16 @@ Release History
 dev
 ---
 
--   \[Short description of non-trivial change.\]
-
```

Comment:
Why did you remove this?

Reply:
There were merge conflicts, and the remote already had that overwritten.